doc/man3/OSSL_STORE_INFO.pod

```diff
@@ -54,25 +54,15 @@ loaders to create B<OSSL_STORE_INFO> holders.
 =head2 Types
 
 B<OSSL_STORE_INFO> is an opaque type that's just an intermediary holder for
-the objects that have been retrieved by OSSL_STORE_load() and similar
-functions.
-Supported OpenSSL type object can be extracted using one of
-STORE_INFO_get0_TYPE().
-The life time of this extracted object is as long as the life time of
-the B<OSSL_STORE_INFO> it was extracted from, so care should be taken not
-to free the latter too early.
```

Comment:
The warning about lifetime should be retained.  Since you removed it here, maybe it can be added back with the description of the get0 functions (feel free to edit it to be suitable)

Reply:
I removed the section about `STORE_INFO_get0_TYPE` and `-STORE_INFO_get1_TYPE` because they do not (any more) exist. Yet I just realized that likely `STORE_INFO_get_type` can be used instead, so I restored the first sentence of that hint, now referring to that function.

Still I see no point in restoring 
```
The life time of this extracted object is as long as the life time of
the B<OSSL_STORE_INFO> it was extracted from, so care should be taken not
to free the latter too early.
```
because its result is not an allocated object but a simple number.

Reply:
> I removed the section about STORE_INFO_get0_TYPE and -STORE_INFO_get1_TYPE because they do not (any more) exist.

What on earth are you talking about?
... OH, I see, you took the upper case `TYPE` literally, instead of just being a placeholder word for the different types of info (`NAME`, `PARAMS`, `PKEY`, `CERT`, `CRL`).

``` console
$ grep OSSL_STORE_INFO_get include/openssl/store.h
 * extracted with OSSL_STORE_INFO_get0_PKEY(), OSSL_STORE_INFO_get0_CERT(), ...
int OSSL_STORE_INFO_get_type(const OSSL_STORE_INFO *info);
const char *OSSL_STORE_INFO_get0_NAME(const OSSL_STORE_INFO *info);
char *OSSL_STORE_INFO_get1_NAME(const OSSL_STORE_INFO *info);
const char *OSSL_STORE_INFO_get0_NAME_description(const OSSL_STORE_INFO *info);
char *OSSL_STORE_INFO_get1_NAME_description(const OSSL_STORE_INFO *info);
EVP_PKEY *OSSL_STORE_INFO_get0_PARAMS(const OSSL_STORE_INFO *info);
EVP_PKEY *OSSL_STORE_INFO_get1_PARAMS(const OSSL_STORE_INFO *info);
EVP_PKEY *OSSL_STORE_INFO_get0_PKEY(const OSSL_STORE_INFO *info);
EVP_PKEY *OSSL_STORE_INFO_get1_PKEY(const OSSL_STORE_INFO *info);
X509 *OSSL_STORE_INFO_get0_CERT(const OSSL_STORE_INFO *info);
X509 *OSSL_STORE_INFO_get1_CERT(const OSSL_STORE_INFO *info);
X509_CRL *OSSL_STORE_INFO_get0_CRL(const OSSL_STORE_INFO *info);
X509_CRL *OSSL_STORE_INFO_get1_CRL(const OSSL_STORE_INFO *info);
```

Reply:
Argh!! So this was just a misunderstanding.
I've restored the original test, but using `<TYPE>` instead of `TYPE` to prevent that confusion and adding the list of potential `TYPE` instances.